### ISSUE TITLE
Controller folder

### ISSUE THREAD

**Issue by theamorn** [vishdhmn/Surveys#2]
it's nice that you make a folder `Controller` but you forget to move n3SurveyCollection into that controller, so others will understand your code easier.

From my point of view, you `Controller` should called `Manager` or `Adapter` your idea is try to put the logic of loading data in here, but in Apple MVC

![screen shot 2560-02-16 at 11 12 03 am](https://cloud.githubusercontent.com/assets/6716483/23007118/c65fa2b8-f438-11e6-9efb-3778f2c29e11.png)

Controller it's mean UIViewController, UITableViewController, UICollectionViewController 
so `Controller` here is misleading a bit, other might misunderstand it.

**Comment by vishdhmn** [comment]
Yes, you are right.  I will change the folder to Manager. Will make it easier for others to understand.
